### PR TITLE
add eslint and prettier

### PR DESCRIPTION
**Pull request by Satya-19**

### CODE REVIEW

File: client/src/components/Login.js [hunk 1]
```diff
@@ -16,12 +16,12 @@ const stringifiedParams = queryString.stringify({
 const googleLoginUrl = `https://accounts.google.com/o/oauth2/v2/auth?${stringifiedParams}`;
 
 export function Login() {
-  console.log(process.env.REACT_APP_CLIENT_ID)
+  console.log(process.env.REACT_APP_CLIENT_ID);
```
Comment (ankitjena) [inline client/src/components/Login.js]: Remove this

File: client/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "prettier"
+    ],
+    "rules": {
+        "prettier/prettier": "error"
+    }
+};
```
Comment (ankitjena) [inline client/.eslintrc.js]: The `.eslintrc.json` and `.eslintrc.js` look the same. We need one right!

**Comment by ankitjena** [comment]
We also need vscode editor settings to autofix on save.

File: client/.eslintrc.js (hunk 2 above)
Comment (Satya-19) [reply client/.eslintrc.js]: okk done

File: client/src/components/Login.js (hunk 1 above)
Comment (Satya-19) [reply client/src/components/Login.js]: okk done

**Comment by ankitjena** [comment]
@Satya-19 Always mention the issue no. in PR description

**Comment by Satya-19** [comment]
Okk, From now onwards I will be careful.

**Pull request merged**